src/frontend/apps/standalone_site/src/features/Authentication/api/usePasswordReset/index.spec.tsx

```diff
@@ -0,0 +1,98 @@
+import { renderHook, WrapperComponent } from '@testing-library/react-hooks';
+import fetchMock from 'fetch-mock';
+import React from 'react';
+import { QueryClient, QueryClientProvider, setLogger } from 'react-query';
+
+import { usePasswordReset } from './index';
+
+const queryClient = new QueryClient({
```

Comment:
Why not moving the instantiation in the `Wrapper` component ? Here you are sharing the same query client between each tests